test/atomic-load-store.ll

```diff
@@ -0,0 +1,35 @@
+; RUN: llvm-as < %s -o %t.bc
+; RUN: llvm-spirv %t.bc -o %t.spv
+; RUN: spirv-val %t.spv
+; RUN: llvm-spirv -to-text %t.spv -o - | FileCheck %s
```

Comment:
I see, that purpose of this patch is to add a missing functionality in forward translation, but does it make sense to also add a test for a reversed translation to double check it?

Reply:
The reverse translation is covered by [test/transcoding/atomics.spt](https://github.com/KhronosGroup/SPIRV-LLVM-Translator/blob/master/test/transcoding/atomics.spt#L44-L45)

Reply:
Though test/transcoding/atomics.spt can be extended with --spirv-target-env="SPV-IR", which I have done in 4104901